redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -296,7 +296,6 @@ internal class LazyListContainerCell(
 
   override fun prepareForReuse() {
     super.prepareForReuse()
-    removeAllSubviews()
```

Comment:
Will this break usage without reuse?

I think we can separately discuss making reuse completely automatic for the lazy layouts, but today we need to support both ways.

Reply:
This should be fine without reuse, since we're already fairly aggressively converted "loaded" cells to placeholders when they're prepared for reuse

Reply:
I think this was unnecessary code!

If `binding` was non-null, `binding.unbind()` will immediately call removeAllSubviews() anyway.

If binding was null, then it didn’t have any subviews to begin with.